game-app/game-core/src/main/java/games/strategy/triplea/delegate/battle/StrategicBombingRaidBattle.java

```diff
@@ -882,14 +882,9 @@ private void findCost(final IDelegateBridge bridge) {
           }
         }
 
-        costThisUnit =
-            Math.max(
-                0,
-                (costThisUnit
-                    + TechAbilityAttachment.getBombingBonus(
-                        attacker.getType(),
-                        TechTracker.getCurrentTechAdvances(
-                            attacker.getOwner(), gameData.getTechnologyFrontier()))));
+        final int bonus =
+            gameData.getTechTracker().getBombingBonus(attacker.getOwner(), attacker.getType());
+        costThisUnit = Math.max(0, (costThisUnit + bonus));
```

Comment:
Ignoring integer overflows, can `costThisUnit` ever be assigned a negative value if the `Math#max` part is omitted?

Reply:
Yes, it can, since the bonuses from tech are just integers so they can be negative.

Reply:
I just feel like"bonus" sounds like a value that's always positive so I wondered